library/Fission/User/Creator/Class.hs

```diff
@@ -88,6 +88,7 @@ instance
       , userEmail         = Just email
       , userRole          = Regular
       , userActive        = True
+      , userVerified      = False
```

Comment:
Hmmm this does make me wonder about having the two tables. Two ideas:

1. Have nullable text field for the current challenge. If it's null, the user has been verified. Write a helper `isVerified`.

2. Make users and emails a one-to-many (user has many email addresses). Keep the challenge on the `EmailAddresses` table.

Thoughts?

Reply:
> Hmmm this does make me wonder about having the two tables
why does this make you wonder about two tables?

I'm down for either for those ideas though. Are we planning to make email addresses one-to-many at some point?

Reply:
Oh I see. Yeah, almost certainly many-to-one down the road, yeah. I asked Bori about this last week, and he said to keep it to whatever is going to be less error prone for a single email for now, since we don’t know the priority of multi-email. Could be a while.